### DISCUSSION TITLE
Consider moving repository to Encode

### DISCUSSION THREAD

**Discussion by ArcLightSlavik** [fastapi/fastapi#7314] [Questions] [answered]
Encode organisation already includes starlette, uvicorn, httpx and database. All of those tools are essential for FastAPI. Also tiangolo is already a member of the organisation. 

In terms of transfer, it seems that github preserves everything. See here: https://docs.github.com/en/github/administering-a-repository/transferring-a-repository. If anyone has experience moving repositories and knowns what it might leave out would be nice to know 🙂

**Comment by Kludex** [comment]
Why?

**Comment by ArcLightSlavik** [comment]
I think it would be better to have stuff that is quite closely aligned together, the Encode org has httpx uvicorn and starlette which makes me feel fastapi would be a good fit there.

**Comment by ghost** [comment]
But is FastAPI Encode's responsibility?

**Comment by tiangolo** [comment, marked as answer]
Thanks, but I'll pass on this.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
